src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -110,7 +110,7 @@ namespace AirLoopHVACDOAS {
 
         if (this->SumMassFlowRate == 0.0 && !state.dataGlobal->BeginEnvrnFlag) {
             state.dataLoopNodes->Node(this->m_CompPointerAirLoopMixer->OutletNodeNum).MassFlowRate = 0.0;
-            return;
+            if (state.dataGlobal->KickOffSimulation) return;
         }
```

Comment:
Shouldn't any report variables that need to be reset be done in initAirLoopDOAS? and then if there is no DOAS flow rate you could return here all the time. Would that work?

    if (this->SumMassFlowRate == 0.0) return;

Reply:
This are report variables of child objects such as fans, heating and cooling coils and are reported by the components themselves. `if (this->SumMassFlowRate == 0.0) return;` is already in the current and did not work.

Reply:
Unless you simulate the child objects (Components on DOAS), report variables of child objects carry over values from the previous time step. I think, we also need to reset flow rates at the nodes of all child objects.